src/main/java/io/anserini/search/ScoredDoc.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Anserini: A Lucene toolkit for reproducible information retrieval research
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.anserini.search;
+
+import org.apache.lucene.document.Document;
+
+public class ScoredDoc {
+  public String docid;
+  public int lucene_docid;
+  public float score;
+  public Document lucene_document;
```

Comment:
trivial comment: it'd be nice to keep Java variable naming conventions (e.g., `docId` instead of `docid`, `luceneDocId` instead of `lucene_docid`, `luceneDocument` instead of `lucene_document`).

Reply:
Ah, I've gone back and forth on this - this class is meant for Pyserini Python bindings... so the options are to use Python conventions in Java or use Java conventions from Python... I opted for the first.

Reply:
ah yes, that's good then !